devicon.json

```diff
@@ -7945,6 +7945,39 @@
         "color": "#bad05e",
         "aliases": []
     },
+    {
+        "name": "v8",
+	    "altnames": [
+            "v8 engine"
+        ],
+        "tags": [
+            "javascript-runtime",
+            "framework",
+            "javascript",
+            "web",
+            "cpp"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#00C4CC",
+	    "aliases": [
```

Comment:
```suggestion
	      "aliases": [
```

Reply:
does aliases is in color??

Reply:
No. I've only suggested in lines 7950 and 7970 two spaces to align your entry. 🙂